content/services/planning.md

```diff
@@ -171,13 +171,13 @@ summary: |
 
   Using an [object-oriented user experience strategy]
```

Comment:
@SondraE Should this be capitalized?

```suggestion
  Using an [Object-Oriented User Experience Strategy]
```

Reply:
@jgerigmeyer Let's consistently not capitalize certified object-oriented user experience strategist, but capitalize UX and OOUX. (I was inconsistent just because I wasn't paying close enough attention.)